pkg/internalregistry/controllers/service_account_controller.go

```diff
@@ -304,10 +311,10 @@ func (c *serviceAccountController) legacyImagePullSecretName(ctx context.Context
 }
 
 var expectedLegacyAnnotations = map[string]func(*corev1.ServiceAccount, string) bool{
-	corev1.ServiceAccountNameKey:                   func(sa *corev1.ServiceAccount, v string) bool { return sa.Name == v },
-	corev1.ServiceAccountUIDKey:                    func(sa *corev1.ServiceAccount, v string) bool { return sa.UID == types.UID(v) },
-	controllers.ServiceAccountTokenSecretNameKey:   func(sa *corev1.ServiceAccount, v string) bool { return true },
-	controllers.ServiceAccountTokenValueAnnotation: func(sa *corev1.ServiceAccount, v string) bool { return true },
+	corev1.ServiceAccountNameKey:       func(sa *corev1.ServiceAccount, v string) bool { return sa.Name == v },
+	corev1.ServiceAccountUIDKey:        func(sa *corev1.ServiceAccount, v string) bool { return sa.UID == types.UID(v) },
+	ServiceAccountTokenSecretNameKey:   func(sa *corev1.ServiceAccount, v string) bool { return true },
+	ServiceAccountTokenValueAnnotation: func(sa *corev1.ServiceAccount, v string) bool { return true },
```

Comment:
Let's use these strings directly:

```suggestion
	"openshift.io/token-secret.name":   func(sa *corev1.ServiceAccount, v string) bool { return true },
	"openshift.io/token-secret.value": func(sa *corev1.ServiceAccount, v string) bool { return true },
```